docs/products/clickhouse/howto/integrate-kafka.rst

```diff
@@ -0,0 +1,164 @@
+``CLICKHOUSE_SERVICE_NAME``      Name of your Aiven for ClickHouse service.
+``KAFKA_SERVICE_NAME``           Name of the Apache Kafka service you use for the integration.
+``PROJECT``                      Name of Aiven project where your services are located.
+``CONNECTOR_TABLE_NAME``         Name of the Kafka engine virtual table that is used as a connector.
+``DATA_FORMAT``                  Input/output data format in which data is accepted into ClickHouse. See :doc:`../reference/supported-input-output-formats`
+``CONSUMER_GROUP_NAME``          Name of the consumer group. Each message is delivered once per consumer group.
+============================     ==========================================================================================================
+
+Create an integration
+----------------------
+
+You can create an integration with the help of :ref:`Aiven CLI <avn_service_integration_create>`, or through the `Aiven console <https://console.aiven.io/>`_ by following these steps:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_, choose the right project, and select your Aiven for ClickHouse service.
+#. In the *Overview* tab, scroll till the section *Service integrations* and click on **Set up integration**. A modal window with available integration options will appear.
+#. Select Apache Kafka from the list of the options.
+#. Choose your service from the list. This is also a place where you can trigger creation of a new Aiven for Apache Kafka service.
+#. Click on **Enable**.
+#. Your Apache Kafka service will be added to the list of enabled service integrations.
+#. You can now close *Service integrations* modal window.
+#. The Kafka integration table will now appear in the database.
+
+.. note::
+
+    Creation of this default configuration only creates an empty ClickHouse but does not create any table. This database is named after the Kafka service.
+
+
+Integration settings
+-----------------------
+
+Next step is to configure the topic and data format options for the integration. This will create a virtual table inside ClickHouse that can receive and send messages from multiple topics. You can have as many of such tables as you need. You need to define for each table
+
+
+* ``name`` - name of the connector table
+* ``columns`` - array of columns, with names and types
+* ``topics`` - array of topics, where you want to bring the data from
+* ``data_format`` - your preferred format for data input, see :doc:`../reference/supported-input-output-formats`
+* ``group_name`` - consumer group name, that will be created on your behalf
+
+Integration settings in a JSON format:
+
+.. code:: json
+
+    {
+        "tables": [
+            {
+                "name": "CONNECTOR_TABLE_NAME",
+                "columns": [
+                    {"name": "id", "type": "UInt64"},
+                    {"name": "name", "type": "String"}
+                ],
+                "topics": [{"name": "topic1"}, {"name": "topic2"}],
```

Comment:
It will going to write from both topics at the same time. And ClickHouse needs a proper data schema, so both topics should have that.

If there is an issue with the schema alignment for Kafka, the data consumption will fail. I don't have a good solution  of how to get out of that error yet (I need to research more what is recommended). As long as the "bad" item remains in the topic, ClickHouse will stop accepting any new items.